Corona-Warn-App/src/main/res/layout/contact_diary_add_location_fragment.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Could we remove `layout`?

Reply:
Yeah, I'll update it. from both location andperson.

Reply:
Looks like I didn't :) I'll format it.